packages/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,233 @@
+import {
+   Flex,
+   Link,
+   Breadcrumb,
+   BreadcrumbItem,
+   Menu,
+   MenuButton,
+   MenuList,
+   MenuItem,
+   MenuItemProps,
+   IconButton,
+   Text,
+   BreadcrumbItemProps,
+   LinkProps,
+} from '@chakra-ui/react';
+import { FaIcon } from '@ifixit/icons';
+import { faChevronRight } from '@fortawesome/pro-solid-svg-icons/faChevronRight';
+import { faEllipsis } from '@fortawesome/pro-solid-svg-icons/faEllipsis';
+import { memo, useState, useCallback } from 'react';
+import isEqual from 'lodash/isEqual';
```

Comment:
This could also be a culprit for the size increase. It's likely this can be dropped.

I was using it to deep equals check props for memoization. This is left over from when I was doing browser width rendering.

Reply:
I checked it, and it didn't seem like it was costing much in size.